articles/android-mdm-setup.md

```diff
@@ -0,0 +1,59 @@
+# Android MDM setup
+
+To turn on Android MDM features, follow the instructions on this page to connect Android Enterprise
+to Fleet.
+
+Fleet supports Android devices that are [Play Protect certified](https://support.google.com/googleplay/answer/7165974?hl=en) (previously known as GMS).
+
+## Turn Android MDM
+
+To turn on Android MDM, connect Android Enterprise on **Settings > Integrations > Mobile device management (MDM)** page.
+
+When you select **Connect Android Enterprise**, Fleet will open the Google sign-up page. The registration process varies depending on whether your organization uses [Google Workspace](#google-workspace), [Microsoft 365](#microsoft-365), or [another provider](#other). Organizations using Google Workspace and Microsoft don't need to verify domain ownership.
+
+### Google Workspace
+
+1. If your organization already uses Google Workspace, use the admin account (or if you don't have one ask your Google Workspace admin to create one for you) to register Android Enterprise.
+2. Follow the steps in Google's sign-up flow to finish the registration.
+3. After successful registration Android Enterprise subscription (free) is added to your Google Workspace, and you can see in Fleet that Android MDM is turned on.
+4. Go to [Google Admin console](https://admin.google.com)
+5. From the side menu, select **Devices > Mobile & endpoints > Settings > Third-party integrations**.
+6. Select **Android EMM**, check **Enable third-party Android mobile management**, then select **Manage EMM providers**.
+7. Toggle **Authenticate Using Google** switch for your Android Enterprise, select the cross icon in the top left corner, and select **Save**.
+
+### Microsoft 365
+
+1. If your organization uses Microsoft 365, you can use your email to register Android Enterprise. When you click **Connect Android Enterprise**, you'll see option to **Sign in with Microsoft**.
```

Comment:
```suggestion
1. If your organization uses Microsoft 365, you can use your email to register Android Enterprise. When you select **Connect Android Enterprise**, you'll see option to **Sign in with Microsoft**.
```

Select instead of click for consistency.